game/forms.py

```
@@ -9,6 +9,12 @@ class CustomUserCreationForm(UserCreationForm):
     class Meta(UserCreationForm.Meta):
         fields = UserCreationForm.Meta.fields + ('email',)
 
+    def clean_email(self):
+        email = self.cleaned_data['email']
+        if self.Meta.model.objects.filter(email__iexact=email).exists():
+            raise ValidationError('A user with this email address already exists.')
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)